packages/devtools_app/lib/src/shared/primitives/utils.dart

```diff
@@ -1214,6 +1214,10 @@ extension ListExtension<T> on List<T> {
     }
     return false;
   }
+
+  T get second => this[1];
+
+  T get third => this[2];
```

Comment:
We should really just add extensions for `first` thru `ninth` or `ninetyninth` if we want to be future proof... 😜